README.md

```diff
@@ -275,3 +278,12 @@ If you want to write an extension for Solidus, you can use the
 
 Solidus is an open source project and we encourage contributions. Please read
 [CONTRIBUTING.md](CONTRIBUTING.md) before contributing.
+
+
+## Sponsors
+
```

Comment:
We are using `Sponsor` here and saying thanks to `bakers` into the text. This is kinda confusing and I think we should explain better the difference between the two.

Reply:
This is true because in OC those are names for the tiers we are using. Backer is $10-50 and Sponsor is $100-500.

Maybe `## Sponsors` can change to `## Key Stakeholders` and perhaps we can say: 
``` 
Thank you to all our donors! 🙏 [Become a donor](https://opencollective.com/solidus)
```

I'd also like to see this higher in the readme to demonstrate the support + provide more value to those willing to donate such a significant amount of money.

Reply:
Agreed and changed. How's that look? https://github.com/seand7565/solidus/tree/readme_oc#key-stakeholders